ecs/cloudwatch.conf

```diff
@@ -0,0 +1,21 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
+
+[OUTPUT]
+    Name   cloudwatch_logs
+    Match  *
+    region us-east-1
+    log_group_name fluent-bit-cloudwatch
+    log_stream_prefix from-fluent-bit-
```

Comment:
these are not the log group and stream names that we want. 

For now let's do:
```
log_group_name /aws/ecs/containerinsights/${ECS_CLUSTER}/application
log_stream_prefix ecs-
```